### PR TITLE
Update slack tips

### PR DESCRIPTION
**Pull request by rmtolmach**
Add section about message retention and add to the Paid Slack Plan section.

Victoria and I made these changes. We're open to feedback and suggestions!

### CODE REVIEW

File: TipsForSlack.md [hunk 1]
```diff
@@ -37,8 +37,26 @@ Please use your best judgement regarding @here and @channel. We don’t want to
...
+
+### Message Retention
+
+Because we are on a free Slack plan, the admins cannot enforce a message and file [retention policy](https://slack.com/help/articles/203457187-Customize-message-and-file-retention). At the time of writing, Slack's official retention policy for free plans is:
+
+> By default, Slack will retain all messages and files (including audio and video clips) for the lifetime of your workspace.
+
+This means that even though we can only view a limited history of messages and files in our free workspace, if the workspace ever moved to a paid plan or if Slack ever increased the free plan message limit, messages that were previously inaccessible would again become accessible and searchable.
+
+If you would like to manually remove your own messages from Slack, you can do so, but you will not be able to institute a personal retention policy using Slack's retention policy settings.
```
Comment (lizrush) [inline TipsForSlack.md]: > If you would like to manually remove your own messages from Slack, you can do so

I had no idea someone could do this! Is there a Slack help article that explains how to do this that we could hyperlink here to explain?

**Review by lizrush — APPROVED** [review]
This looks great to me! I know you requested feedback in slack so I will approve the PR (based on my own opinions only 🙂) but will wait to take any action until you let me know we're ready so we can give others an opportunity for input!

File: TipsForSlack.md (hunk 1 above)
Comment (laneia) [reply TipsForSlack.md]: I could be wrong but I think through the UI you can only do it 1 by 1? (As long as deleting is not disabled by workspace admin, in which case you have to edit over prior messages.)

The API can be leveraged for a faster solution:
https://api.slack.com/methods/chat.delete

And I think this is the more up to date repo in python that could be used for bulk delete too:
https://github.com/sgratzl/slack_cleaner2

Comment (a-lmx) [inline TipsForSlack.md]: +1 to specifying that messages have to be deleted individually.

Comment (rmtolmach) [reply TipsForSlack.md]: Victoria had done some looking into this and found a chrome extension as well. How about this?

> If you would like to manually remove your own messages from Slack, you can do so, but you will not be able to institute a personal retention policy using Slack's retention policy settings. Some options for deleting your messages include manually clicking "Delete message" for each message in the UI, using the [Slack API](https://api.slack.com/methods/chat.delete), or leveraging a Chrome extension.

**Comment by rmtolmach** [comment]
Made changes. Merging!

**Pull request merged**